docs/user-guide/configuration/sourcePaths.md

```diff
@@ -48,6 +48,10 @@ jobs:
 
 In this example, the job `main` will be triggered if there are any changes to files under `src/app/` or the `screwdriver.yaml` file (like on `src/app/main.js`, `src/app/package.json`, etc.). The `main` job will **not**, however, be triggered on changes to `README.md`, `test/`, or `src/other/`.
 
+### Matched source path
+
+Screwdriver will expose the matched souce path in an environment variable `SD_SOURCE_PATH`. This can be used to write scripts which depends on the source path that triggered the build.
```

Comment:
Based on the logic in model, SD_SOURCE_PATH will be the FIRST match path. It is possible that a PR can multiple files match different source defined. Should we say anything about that in the doc?

Reply:
Should it return *all* matches vs just the *first*?